.agents/skills/release-validation/scripts/collect-release-inventory.py

```
@@ -0,0 +1,214 @@
+            "remote and rerun"
+        ) from error
+
+
+def collect_commits(base: str, head: str) -> list[dict]:
+    delimiter = "\x1f"
+    record = "\x1e"
+    output = run(
+        "git",
+        "log",
+        "--reverse",
+        f"--format=%H%x1f%aI%x1f%an%x1f%ae%x1f%s%x1f%b%x1e",
```

Ruff (0.16.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)